account_analytic_project/model/account_analytic_account.py

```diff
@@ -0,0 +1,77 @@
+# -*- encoding: utf-8 -*-
+"""Extend account.analytic.account."""
```

Comment:
I agree with @lmignon: not a blocking issue, but these docstrings come unexpected and I would prefer to not have them